sql-ballerina/Module.md

```diff
@@ -189,12 +216,12 @@ error? e = resultStream.forEach(function(Student student) {
 ```
```

Comment:
Since the only change is closed record type. others are the same. let's add only one code sample

Reply:
SQL Query is also changed. that is why I added another code sample.

Reply:
yes, let's remove it. the sample doesn't reflect our intention clearly.